CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ This is a history of changes to clara-rules.
 # 0.21.0 SNAPSHOT
 * Add names to anonymous functions generated by rule compilation, see [issue 261](https://github.com/cerner/clara-rules/issues/261) and [issue 291](https://github.com/cerner/clara-rules/issues/291)
 * Add alpha node types, see [issue 237](https://github.com/cerner/clara-rules/issues/237)
+* BREAKING: Validate parameters provided to queries exist on query and compilation. see [issue 454](https://github.com/cerner/clara-rules/issues/454)
```

Comment:
It may be better to not just call it breaking. It is in a sense, but previously you’d get no result basically. Perhaps mention? Or is the issue link enough I guess?

Reply:
+1 to the changes in https://github.com/cerner/clara-rules/pull/455/commits/b559d93351f34cd058a05435b3a32dafa926bcb9